save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -89,7 +89,7 @@ data class GeneralConfig(
             this.excludedTests ?: other.excludedTests,
             this.expectedWarningsPattern ?: other.expectedWarningsPattern,
             this.runConfigPattern ?: other.runConfigPattern,
-        )
+        ).also { it.configLocation = this.configLocation }
```

Comment:
Can we make `configLocation` a constructor parameter, so that we don't forget to add `also` in the future? And generally everything would look more consistent

Reply:
I think the main thing, why we initially took out it from ctor, because of serialization and ability to provide it in `toml` files - but this field is used only for internal purposes and shouldn't be available from outside.

On the other hand - it's really easy to forget about updating of this field in case of creation of a new instance, like we did. However, the only `*PluginConfig.kt` classes for now are responsible for configs creation, and it seems, that we will not expand this classes a lot in the near future.

I would better leave this field as is, at least for now.

Reply:
Ok, let's leave it as is